Library/Formula/hspell.rb

```diff
@@ -0,0 +1,19 @@
+class Hspell < Formula
+  desc "The Hspell project is a free Hebrew linguistic project."
+  homepage "http://hspell.ivrix.org.il/"
+  url "http://hspell.ivrix.org.il/hspell-1.3.tar.gz"
+  sha256 "603c25dcbaa65d171d9065da7369cfe0dc21bda8378bade13b42eda69c8b2fe7"
+
+  def install
+    inreplace "Makefile.in", "soname", "install_name"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!